src/main/resources/db/migration/h2/V1.13__organisation-reporting-permission.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS `civil_servant_organisation_reporting_permission`;
```

Comment:
DROP TABLE should not be included here as it may lead to unexpected results.

Reply:
will remove.